addons/l10n_pl/__manifest__.py

```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 # Copyright (C) 2009 - now Grzegorz Grzelak grzegorz.grzelak@openglobe.pl
 
 {
     'name' : 'Poland - Accounting',
     'version' : '2.0',
-    'author' : 'Grzegorz Grzelak (OpenGLOBE)',
+    'author': ['Odoo S.A.', 'Grzegorz Grzelak (OpenGLOBE)'],
```

Comment:
First time I see multiple authors in a list like this... I think you need to do that instead:
```suggestion
    'author': "Odoo S.A.,Grzegorz Grzelak (OpenGLOBE)",
```